README.md

```diff
@@ -1,4 +1,3 @@
-# morgan
```

Comment:
Why are you remocing the module name from the top? This is how we format all our expressjs readmes.